pipeline/snapcraft/snapcraft.go

```diff
@@ -48,13 +54,15 @@ func (Pipe) Description() string {
 
 // Run the pipe
 func (Pipe) Run(ctx *context.Context) error {
-	if ctx.Config.Snapcraft.Summary == "" {
-		log.Error("no snapcraft summary defined, skipping")
+	if ctx.Config.Snapcraft.Summary == "" && ctx.Config.Snapcraft.Description == "" {
+		log.Warn("skipping because no summary nor description were provided")
```

Comment:
yeah, I'll make that in another patch. I'm thinking about creating a global `SkipError` or something like that...